templates/firstbyte/base.html

```diff
@@ -152,7 +119,7 @@ <h6 class="footer-nav-header mb-0">ADDITIONAL LINKS</h6>
                         <span class="text-muted px-2">|</span>
                         <span class="text-muted"><a href="https://github.com/lukesanantonio/firstbyte" target="blank">View project on Github</a></span>
                     </div>
-                </div>
+                </div>  
```

Comment:
I know it's nitpicky :smile: but could you trim whitespace if possible? (Usually your editor will have a setting)